apps/asap-cli/src/import/protocols/insert.ts

```diff
@@ -83,7 +87,7 @@ const addProtocolToTeam = async (
 
 export default async (data: Protocol): Promise<void> => {
   const promises: Cache = {};
-  const protocol: RestResearchOutput['data'] = {
+  const protocol: Omit<RestResearchOutput['data'], 'pmsEmails'> = {
```

Comment:
actually you could swap it for the new CreateResearchOutput type

Reply:
there's no `type` field on `CreateResearchOutput`, it complains